app/views/RoomInfoView/components/UserInfoIcon.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Text } from 'react-native';
+import { BorderlessButton } from 'react-native-gesture-handler';
+
+import { CustomIcon, TIconsName } from '../../../containers/CustomIcon';
+import styles from '../styles';
+import { useTheme } from '../../../theme';
+import { useVideoConf } from '../../../lib/hooks/useVideoConf';
+import i18n from '../../../i18n';
+import { useAppSelector } from '../../../lib/hooks';
+import { compareServerVersion } from '../../../lib/methods/helpers';
+
+// TODO: change other icons on future
+export function UserInfoIcon({
```

Comment:
I find this `*Icon` naming awkward as well.
Ok, it has an icon, but it really is a button, because the goal for the component is to be pressed. Not to merely display something.

It's also odd, because this can happen:
```tsx
<UserInfoIcon showIcon={false} />
```

An icon that has no icon.